kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -113,6 +113,14 @@ def wiki_to_html(wiki_markup, locale=settings.WIKI_DEFAULT_LANGUAGE,
                                              nofollow=nofollow))
 
 
+@library.filter
+def wiki_to_safe_html(wiki_markup, locale=settings.WIKI_DEFAULT_LANGUAGE,
```

Comment:
This filter needs to be able to strip the disallowed tags from the final output. 

Here's what I get when saving a test profile.

![image](https://user-images.githubusercontent.com/584352/36835522-2e598642-1d3f-11e8-982d-cb6f928e6844.png)

instead of 

![image](https://user-images.githubusercontent.com/584352/36835542-42b0eec8-1d3f-11e8-9dcd-542d76797e6b.png)

Reply:
Ah... hmm... I thought bleach was stripping tags. I was going to add more bleach processing to the output of `wiki_to_html` but discovered that it's already doing bleach.clean, just with a lot more tags. I can look at making it strip tags instead of converting them to HTML entities. It's also possible we don't need to worry too much since people can just edit again to remove the HTML.